net.odamex.Odamex.yml

```diff
@@ -0,0 +1,76 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+---
+app-id: net.odamex.Odamex
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: select-exe
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --device=all # change this to device=input + device=dri when available more widely
+  - --socket=pulseaudio
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=host
+  - --env=TIMIDITY_CFG=/app/etc/timidity.cfg
+modules:
+  - name: timidity-instruments
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/etc/
+      - install timidity.cfg /app/etc/sdl_timidity.cfg
+      - mkdir -p /app/share/timidity/
+      - cp -r instruments /app/share/timidity/instruments
+    sources:
+      - type: archive
+        url: http://www.libsdl.org/projects/mixer/timidity/timidity.tar.gz
+        sha256: ee7c59bb5d5fae206d53eef0e82b5ed739e7cbf69252a3754024f8f52ebe77aa
+  - name: SDL2-mixer
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -D SDL2MIXER_VENDORED=ON
+      - -D SDL2MIXER_DEPS_SHARED=OFF
+      - -D SDL2MIXER_SAMPLES=OFF
+      - -D CMAKE_INSTALL_LIBDIR=lib
+    sources:
+      - type: git
+        url: https://github.com/libsdl-org/SDL_mixer.git
+        tag: release-2.8.1
+  # cpptrace dependencies
+  - name: zstd
```

Comment:
this is in the runtime.

Reply:
SDL mixer? or zstd? The SDL mixer included with the runtime is built without support for MOD or MIDI

Reply:
I didn't know there was ambiguity 

![image](https://github.com/user-attachments/assets/3ad462a7-2ba6-4d6f-bd9c-1dee55ebf1ef)